### PR TITLE
Suggest earlier weeks/subjects as input for new scripts

### PR DESCRIPTION
**Pull request by rpkamp**
screenshot from 2017-05-08 09-51-21

### CODE REVIEW

**Comment by seggen-ibuildings** [comment]
Have you considered using something like: http://ioncache.github.io/Tag-Handler/ or http://bootstrap-tagsinput.github.io/bootstrap-tagsinput/examples/?

File: web/js/admin.js [hunk 1]
```diff
@@ -74,6 +74,13 @@ var admin = (function ($, chartOptions) {
                 }
             });
         });
+
+        $('body').on('click', '.form-suggestions button', function (e) {
+            var $target = $('#' + $(this).data('suggestion-target'));
+            $target.val($target.val() + $(this).text());
```
Comment (seggen-ibuildings) [inline web/js/admin.js]: If you don't want multiple values, why not replace the value?
And maybe select the contents, so you can start typing right away to correct the thing you automatically inserted...

Comment (rpkamp) [inline web/js/admin.js]: Replacing is probably better, yes. I'll change it.

Comment (rpkamp) [inline web/js/admin.js]: Fixed. Please review again :)

**Pull request merged**